docker-compose.yml

```diff
@@ -138,126 +138,70 @@ services:
       options:
         max-size: '10m'
 
-  zookeeper:
-    image: wurstmeister/zookeeper
-    ports:
-      - '2181:2181'
-    logging:
-      options:
-        max-size: '10m'
-        max-file: '5'
-
   kafka:

```

Comment:
I'm seeing a warning message when running `docker compose up -d` with "the attribute 'version' is obsolete". While we're in this file, let's get it up to spec too

Reply:
removed